go/vt/mysqlctl/mycnf_test.go

```diff
@@ -86,7 +86,6 @@ func TestMycnf(t *testing.T) {
 // 3. go test
 // 4. \rm $VTROOT/vthook/make_mycnf
 // 5. Add No Prefix back
-//lint:ignore U1000 Test for Mycnf hook changes
```

Comment:
Why are these directives being removed? I don't even know what they are, but I don't see the connection to this PR.